survey-resources/data-xml/demo-survey-v1.xml

```diff
@@ -161,6 +161,9 @@
           <text id="/arcEm5iPB4F9CQZR258k4r/group_pa5ah98/Do_you_own_or_rent_your_home/rent:label">
             <value>Alquilado</value>
           </text>
+          <text id="/arcEm5iPB4F9CQZR258k4r/group_pa5ah98/Do_you_own_or_rent_your_home/rent:label">
+            <value>Ninguno</value>
+          </text>
```

Comment:
also you need to change the text for the label, so this needs to be `....home/neither:label`
and then you need to add a new select option for `neither`